src/templates/components/topTasks/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { VamcEhr } from '@/types/drupal/vamcEhr'
+
+type VamcEhrSystem = VamcEhr['field_region_page']['field_vamc_ehr_system']
+
+type TopTasksProps = {
+  path: string
+  vamcEhrSystem: VamcEhrSystem
+  administration?: { entityId: number }
+  regionPage?: { vamcEhrSystem: VamcEhrSystem }
+  office?: { vamcEhrSystem: VamcEhrSystem }
+}
+
+/**
+ * Display a set of actionable links for a facility.
+ *
+ * This is used for Facility details pages and (A-Z) Health Services page.
+ */
+export const TopTasks = ({
+  path,
+  administration,
+  vamcEhrSystem,
+  regionPage,
+  office,
+}: TopTasksProps) => {
+  // Make sure we have exactly one / at the start of the path to avoid malformed
+  // URLs.
+  const slashPath = path.startsWith('/') ? path : `/${path}`
+
+  const topTask = topTaskLovellComp({
+    linkType: 'make-an-appointment',
+    path: slashPath,
+    administration,
+    vamcEhrSystem,
+    regionPage,
+    office,
+  })
+
+  return (
+    <div className="usa-grid usa-grid-full vads-u-margin-bottom--6">
+      <div data-template="facilities/facilities_health_services_buttons">
+        <va-link-action
+          class="vads-u-display--block"
+          href={`${topTask.url}`}
+          text={`${topTask.text}`}
+          type="secondary"
+        ></va-link-action>
+        <va-link-action
+          class="vads-u-display--block"
+          href={`${slashPath}/register-for-care`}
+          text="Register for care"
+          type="secondary"
+        ></va-link-action>
+        <va-link-action
+          class="vads-u-display--block"
+          href={`${slashPath}/pharmacy`}
+          text="Learn about pharmacy services"
+          type="secondary"
+        ></va-link-action>
+      </div>
+    </div>
+  )
+}
+
+/**
+ * Generates a text and URL object for the top task component based on the
+ * provided properties.
+ *
+ * NOTE: May need to export this from another module later to be used elsewhere.
+ */
+const topTaskLovellComp = ({
+  linkType,
+  path,
+  administration,
+  vamcEhrSystem,
+  regionPage,
+  office,
+}: TopTasksProps & { linkType?: 'make-an-appointment' }): {
+  text: string
+  url: string
+} => {
+  // TODO: Is this right?
+  const isProd = process.env.APP_ENV === 'prod'
+
+  return _topTaskLovellComp({
+    isProd,
+    linkType,
+    path,
+    administration,
+    vamcEhrSystem,
+    regionPage,
+    office,
+  })
+}
+
+/**
+ * Same as `topTaskLovellComp` but with an additional `isProd` flag to aid
+ * testing.
+ */
+export const _topTaskLovellComp = ({
```

Comment:
Just a little dependency injection to test that `isProd` flag without fiddling with environment variables in Jest. I'm not married to this approach, though, so if there's a Less Gross™ way, let me know! (I don't think this is awful, but it _is_ a bit weird.)